timeline-chart/src/time-graph-font-controller.ts

```diff
@@ -5,9 +5,10 @@ export class FontController {
     private fontNameMap: Map<number, Map<string, string>>;
     private fontColorMap: Map<number, string>;
     private defaultFontName: string = "LabelFont8White";
+    private minFonSize: number = 6;
```

Comment:
We can make this a local variable as we just use it in one function. Also make it constant.

nit: MIN_FONT_SIZE (variable name)

Reply:
Sure. I will change the name.